Tests/Orso.Arpa.Api.Tests/IntegrationTests/Shared/TestSeed.cs

```diff
@@ -112,5 +113,10 @@ private static async Task SeedUsersAsync(ArpaUserManager userManager, SignInMana
                 await signInManager.CheckPasswordSignInAsync(lockedOutUser, "wrongPassword", true);
             }
         }
+
+        private static async Task SeedTranslationData(IArpaContext arpaContext)
+        {
+            await arpaContext.Translations.AddRangeAsync(Orso.Arpa.Tests.Shared.TestSeedData.TranslationSeedData.Translations);
```

Comment:
Die Daten werden doch bereits in der Migration geseedet? Warum an dieser Stelle nochmal?

Reply:
Ist tatsächlich inzwischen nicht notwendig.
Wurde entfernt.